README.md

```diff
@@ -113,6 +111,8 @@ In organisaties waarin de gebruikers van veg2hab geen volledige local admin rech
 
 ## Gebruikershandleiding
 
+Voor een uitgebreide gebruikershandleiding, zie [Handleiding voor gebruikers](./docs/HANDLEIDING.md).
```

Comment:
HANDLEIDING.md is een subset van wat er hier staat toch? (en dus niet uitgebreider)